deploy/scripts/src/mobile/utils/functions-mobile-journey.ts

```diff
@@ -48,70 +29,65 @@ function isStatusCode302 (res: Response): boolean {
   })
 }
 
-function isPageContentCorrect (res: Response, pageContent: string): boolean {
+function isPageRedirectCorrect (res: Response, pageUrl: string): boolean {
```

Comment:
See if we can absorb this function and isRedirectUrlCorrect into a single function

Reply:
These functions test two different things (page URL vs header location). Have left as two separate functions